src/HomeScreen/components/BaniList/BanilList.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { FlatList } from "react-native";
+import { ListItem } from "@rneui/themed";
+import PropTypes from "prop-types";
+import constant from "../../../common/constant";
+import { useSelector } from "react-redux";
+import { baseFontSize } from "../../../common/helpers";
+
+const renderBanis = (item, navigate, fontSize, fontFace, isTransliteration) => {
```

Comment:
It is a good practice to define a helper function inside the function component if it is being called only in the function component and nowhere else. In this case, I think, `renderBanis` function should be defined inside the `BaniList` function.